### DISCUSSION TITLE
5 languages / bugfixes

### DISCUSSION THREAD

**Discussion by Norbas7** [mechano/Kingroon_KP3S_Pro_Marlin#17] [General]
Can you write what languages ​​are available? Is it Polish?  Where can I find what bugfixes have been made to the version from last time/ year?

**Comment by mechano** [comment]
There's no Polish sorry. But there's all the necessary to add by yourself and compile. 

The bug fixes you can find into official Marlin github's project page. 
Thi firmware is update to 08 september 2024. 

Soon I'll do another updated release.
